server/src/interfaces.ts

```diff
@@ -0,0 +1,172 @@
+/**
+ * Task describes a single action
+that should be done by the user. For example: “Go for a 10
+minutes run” or “Go to bed before 11:00 PM”
+*/
+interface Task {
+  id: number;
+  description: string;
+}
+
+/**
+ * Achievement describes a set of
+several tasks accomplished in
+the specific way. For example:
+“Complete each task 7 days in
+a row” or “Complete 5 tasks
+before 8:00 AM”
+ */
+interface Achievement {
+  id: number;
+  description: string;
+  image: string;
+  checkComplete: boolean;
+}
+
+/**
+ * Status describes a state of
+some item (a task or an
+achievement) and a
+timestamp, when this state
+was updated
+ */
+interface Status {
+  state: 'Pending' | 'Success' | 'Failure';
```

Comment:
Can create an `enum StatusState { PENDING: 'Pending' ... }` elsewhere and than 
```suggestion
  state: StatusState;
```